L1Trigger/L1THGCal/interface/HGCalTriggerCellCalibration.h

```diff
@@ -2,31 +2,34 @@
 #define __L1Trigger_L1THGCal_HGCalTriggerCellCalibration_h__
 
 #include <cstdint>
-#include "FWCore/ParameterSet/interface/ParameterSet.h"
-#include "FWCore/Framework/interface/ESHandle.h"
 #include "DataFormats/Common/interface/Handle.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "Geometry/HGCalCommonData/interface/HGCalDDDConstants.h"
 #include "Geometry/HGCalGeometry/interface/HGCalGeometry.h"
 #include "Geometry/Records/interface/IdealGeometryRecord.h"
-#include "Geometry/HGCalCommonData/interface/HGCalDDDConstants.h"
 
 #include "DataFormats/L1THGCal/interface/HGCalTriggerCell.h"
 #include "L1Trigger/L1THGCal/interface/HGCalTriggerTools.h"
 
 class HGCalTriggerCellCalibration {
-public:
+ public:
   HGCalTriggerCellCalibration(const edm::ParameterSet& conf);
   void eventSetup(const edm::EventSetup& es) { triggerTools_.eventSetup(es); }
   void calibrateInMipT(l1t::HGCalTriggerCell&) const;
   void calibrateMipTinGeV(l1t::HGCalTriggerCell&) const;
   void calibrateInGeV(l1t::HGCalTriggerCell&) const;
   void print();
 
-private:
+ private:
   double LSB_silicon_fC_;
   double LSB_scintillator_MIP_;
   double fCperMIP_;
+  double fCperMIPnose_;
```

Comment:
Is there a reason that the fC per MIP could be different for the HF nose? I thought the same modules were used.

Reply:
All these parameters change on the geometry (HFnose will stay with the V9 for a while). I prefer to be independent on these calibration inputs. I think we can adjust later.

Reply:
ok